src/orchard/classpath.clj

```diff
@@ -51,3 +53,21 @@
    (classpath-jarfiles (classpath)))
   ([path]
    (map #(JarFile. ^File %) (filter cp/jar-file? path))))
+
+(defn classpath-file-relative-path
```

Comment:
Don't we have a patch about this in the `classpath` function already?

Reply:
I will double check but this was in the `info` middleware in `cider-nrepl` and it's been ported over here

Reply:
Not sure about what is going on in this function actually, so I would rather leave it here for now cause it can break functionality to change it. I can open a separate issue when merged to track it.